managed/src/main/java/com/yugabyte/yw/controllers/EncryptionAtRestController.java

```diff
@@ -245,13 +222,9 @@ public Result removeKeyRefHistory(UUID customerUUID, UUID universeUUID) {
                 customerUUID.toString(),
                 universeUUID.toString()
         ));
-        try {
-            keyManager.cleanupEncryptionAtRest(customerUUID, universeUUID);
-            Audit.createAuditEntry(ctx(), request());
-          return YWSuccess.asResult("Key ref was successfully removed");
```

Comment:
You are overwriting my YWSuccess change in the merge. Also check your other resolved conflicts to make sure same is not happening elsewhere.

Reply:
Will create separate PR to address merge conflicts.